internal/appconfig/validation.go

```diff
@@ -275,6 +275,14 @@ func (cfg *Config) validateMounts() (extraInfo string, err error) {
 			}
 		}
 
+		switch {
+		case m.SnapshotRetention == 0:
+			m.SnapshotRetention = 5
+		case m.SnapshotRetention < 1 || m.SnapshotRetention > 60:
+			extraInfo += fmt.Sprintf("mount '%s' has a snapshot_retention value which is not between 1 and 60 days\n", m.Source)
```

Comment:
Unless you specifically want single-quotes, you can also use `%q` to print a quoted value.

Reply:
thanks for the tip! all the other validation errors were formatted like this so I'll to keep the format consistent.